### DISCUSSION TITLE
Help Adding Custom Text to Language

### DISCUSSION THREAD

**Discussion by aggieflash11** [atom/atom#23197] [Q&A]
I'm new to atom and programming as a whole. I'm currently coding using the built in html syntax but wanted to see if I could add custom text to the different categories. Could doggos be colored the same as style? Or div?

Sorry if I'm not using the correct terminology, I'm learning it on my own. Thanks!

**Comment by icecream17** [comment]
You can change the color of some things with https://flight-manual.atom.io/using-atom/sections/basic-customization/#style-tweaks
The docs don't really explain how to "inspect element" so if you don't know what that means see here: https://github.com/atom/atom/discussions/22147#discussioncomment-586150

Not sure what you mean by doggos
If you want the html parser to parse extra syntax then that's more complicated...
